htdocs/product/class/product.class.php

```diff
@@ -3477,7 +3477,7 @@ function load_virtual_stock()
             if ($result < 0) dol_print_error($this->db,$this->error);
             $stock_commande_fournisseur=$this->stats_commande_fournisseur['qty'];
 
-            $result=$this->load_stats_reception(0,'4');
+            $result=$this->load_stats_reception(0,'3,4');
```

Comment:
Status 3 is "process running" nothing received yet.
4 is "Received partially".
May be the problem is that with 3.9 (and may be other version), when you record one reception, the status of the supplier order is not set from 3 to 4 ? Can you check that ?